src/integrations/Matomo/browser.js

```diff
@@ -55,19 +55,7 @@ class Matomo {
 
   loadScript() {
     window._paq = window._paq || [];
```

Comment:
This should also be part of the loader script.

Reply:
Updated